subt_ign/scripts/playback_event_recorder/playback_event_recorder.sdf

```diff
@@ -0,0 +1,104 @@
+<?xml version='1.0'?>
+<!--
+  Playback event recorder.
+
+  This world is used by the record_playback_events.bash script.
+  Videos will be saved to a timestamped directory inside the directory
+  where the bash script is run.
+
+  To set whether or not the gui camera should follow the entity
+  during video recording, change the <camera_follow> sdf parameter in the
+  PlaybackEventRecorder plugin to either true or false.
+
+  Usage: bash record_playback_events.bash <path_to_log_dir>
+-->
+
+<sdf version='1.6'>
+
+  <world name='default'>
+    <gui fullscreen="0">
```

Comment:
Is it safe to remove the `<gui>` section? I'm trying to setup 1080p recording, and I can't seem to get the window to use the `gui.config` when the `<gui>` tag is in place.

Reply:
yeah I just commented it out in 35acfe7. I added the gui section mainly to remove the entity tree and component inspector plugins that are enabled by default.